bundle-workflow/src/getfiles_S3bkt.py

```diff
@@ -0,0 +1,36 @@
+# !/usr/bin/env python3
```

Comment:
Name this file as `get_files_from_S3` or something similar following the same naming convention